pkg/cri/server/container_execsync.go

```diff
@@ -72,6 +73,9 @@ func (cw *cappedWriter) isFull() bool {
 // ExecSync executes a command in the container, and returns the stdout output.
 // If command exits with a non-zero exit code, an error is returned.
 func (c *criService) ExecSync(ctx context.Context, r *runtime.ExecSyncRequest) (*runtime.ExecSyncResponse, error) {
+	span := tracing.SpanFromContext(ctx)
+	span.SetAttributes(tracing.Attribute("container.id", r.GetContainerId()))
```

Comment:
Can `tracing.SpanFromContext()` also accept arbitrary attributes?

Reply:
I actually prefer `SpanFromContext(ctx)` to return the existing span as is from context. This way it is more readable and clear what each api method is used for.